.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -3,7 +3,12 @@ name: ViewComponent bug report
 about: Create a report to help us improve ViewComponent
 ---
 
-<!-- If possible, please provide a PR with a failing test for bugs instead of filing an issue. -->
+<!--
+When opening an issue to describe a bug you're facing, it's helpful to replicate
+the bug. You can do this either by contributing failing tests in a pull request,
+or by creating a new repository that demonstrates the issue. Check out
+CONTRIBUTING.md to get started.
```

Comment:
As a general rule, I think we should let folks' editors handle line wrapping:

```suggestion
When opening an issue to describe a bug, it's helpful to provide steps to reproduce it, either with failing tests in a pull request, or by sharing a repository that demonstrates the issue. See CONTRIBUTING.md for more details.
```

Reply:
Agreed for the general case, but this template will mostly be seen from within the GitHub UI in a `textarea`, I'd assume. At a standard width, that'd look as follows:

<img width="799" alt="Screenshot 2021-12-13 at 20 24 42" src="https://user-images.githubusercontent.com/20680337/145883268-d44d5431-fc00-48a9-9d20-a3905a43e86f.png">

We're already hard-wrapping in the PR template as shown above, so I did the same here under the understanding that it'd feel a bit more readable at that width.

Reply:
I see. I still think that should leave wrapping up to the client. If someone views this on a screen narrower than the wrapping length you've chosen here it will wrap in an odd way.

I recognize that we don't follow this guidance elsewhere and I'd be happy to clean up other violations :)